tests/unit/test_log.py

```diff
@@ -25,32 +27,18 @@ def test_request_log(client, email_factory, caplog):
     assert resp.status_code == 200
     assert len(caplog.records) == 1
     log = caplog.records[0]
-    assert hasattr(log, "duration_s")
-
-    # remove Sentry headers with random trace ids
-    assert log.headers.pop("baggage")
-    assert log.headers.pop("sentry-trace")
 
     expected_log = {
         "client_allowed": True,
-        "client_host": "testclient",
         "client_id": "test_client",
-        "duration_s": log.duration_s,
-        "msg": f"testclient:50000 test_client 'GET /ctms/{email_id} HTTP/1.1' 200",
-        "headers": {
-            "host": "testserver",
-            "user-agent": "testclient",
-            "accept-encoding": "gzip, deflate",
-            "accept": "*/*",
-            "connection": "keep-alive",
-            "x-request-id": "foo-bar",
-        },
+        "uid": "test_client",
+        "agent": "testclient",
         "method": "GET",
         "path": f"/ctms/{email_id}",
-        "path_params": {"email_id": email_id},
-        "path_template": "/ctms/{email_id}",
-        "status_code": 200,
+        "code": 200,
+        "lang": None,
         "rid": "foo-bar",
+        "t": FuzzyAssert(lambda x: x > 0, name="uint"),
```

Comment:
Any reason in particular you didn't want to pull the duration directly off the log object?

Reply:
Like `"t": log.duration` ?

It was a way to really test the value. Like if `log.duration` is None then the assertion will be true

Reply:
Ahh right 🤦🏻